<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>9500000</width>
    <height>5000000</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical {
	border: none;
    background: rgb(40, 84, 113);
    width: 8px;
    margin: 0px 0 0px 0;
	border-radius: 4px;
 }
 QScrollBar::handle:vertical {	
	background: rgb(129,194,65);
    min-height: 25px;
	border-radius: 4px
 }
 QScrollBar::add-line:vertical {
     height: 0px;
}
 QScrollBar::sub-line:vertical {
	height: 0px;
}
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     background: none;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }
QScrollBar:horizontal {
	border: none;
    background: rgb(40, 84, 113);
    height: 8px;
    margin: 0px 0 0px 0;
	border-radius: 4px;
 }
 QScrollBar::handle:horizontal {	
	background: rgb(129,194,65);
    min-width: 25px;
	border-radius: 4px
 }

 QScrollBar::add-line:horizontal {
     width: 0px;
}
 QScrollBar::sub-line:horizontal {
	width: 0px;
}
 QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal {
     background: none;
 }
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
     background: none;
 }

QProgressBar::handle:horizontal {	
	background: rgb(129,194,65);
    min-width: 25px;
	border-radius: 4px
 }

#frame_2 {
  	border-top: 0.5px solid rgb(211,211,211);
	border-bottom:0.5px solid rgb(211,211,211);
 	background-color: rgb(245, 248, 251);
}
#frame_3, #label {	
	background-color: rgb(244,245,235);
}

#frame_5 {
background-color: rgb(40, 84, 113);
}
#label_5 {
	font: 40pt;
	color: rgb(251, 255, 255);
}

#data_app, #tech_app, #eval_app, #btm_app, #perf_app, #eq_app, #micr_app, #plan_app {
	border: 0px solid rgb(40, 84, 113);
	border-radius: 6px;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 15px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="home_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>46</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {	
	border: none;
	background-color: transparent;
	padding-left: 0px;
}
QPushButton:hover {
	border: 2px solid rgb(129, 194, 65);
	border-radius: 4px;
}
QPushButton:pressed {	
	background-color: rgb(129, 194, 65);
	border-radius: 4px;
}
QPushButton:checked {	
	background-color: rgb(129, 194, 65);
	border-radius: 4px;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../resources.qrc">
              <normaloff>:/logos/images/logo/Quest_App_Icon_50_transparent.png</normaloff>
              <normalon>:/logos/images/logo/Quest_App_Icon_50_transparent.png</normalon>:/logos/images/logo/Quest_App_Icon_50_transparent.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>44</width>
              <height>48</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color: rgb(129, 194, 65);
background-color: rgb(40, 84, 113);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;QuESt Planning&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="top_right_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{background-color: rgb(40, 84, 113);}
QPushButton {	
	background-color: rgb(245, 245, 245);
	border-radius: 4px;
}
QPushButton:hover {
	background-color: rgb(129, 194, 65);
	border-radius: 4px;
}
QPushButton:pressed {	
	background-color: transparent;
	border-radius: 4px;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="min_resize_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Minimize&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../resources.qrc">
                 <normaloff>:/icon/images/icons/remove_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/icon/images/icons/remove_FILL0_wght200_GRAD0_opsz24.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="norm_resize_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Restore Down&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../resources.qrc">
                 <normaloff>:/icon/images/icons/open_in_new_FILL0_wght200_GRAD0_opsz48.png</normaloff>
                 <normalon>:/icon/images/icons/apps_FILL0_wght700_GRAD-25_opsz20.png</normalon>
                 <selectedon>:/icon/images/icons/check_box_outline_blank_FILL0_wght200_GRAD0_opsz24.png</selectedon>:/icon/images/icons/open_in_new_FILL0_wght200_GRAD0_opsz48.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="max_resize_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Maximize&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../resources.qrc">
                 <normaloff>:/icon/images/icons/open_in_new_down_FILL0_wght200_GRAD0_opsz48.png</normaloff>
                 <normalon>:/icon/images/icons/open_in_new_down_FILL0_wght200_GRAD0_opsz48.png</normalon>
                 <selectedon>:/icon/images/icons/check_box_outline_blank_FILL0_wght200_GRAD0_opsz24.png</selectedon>:/icon/images/icons/open_in_new_down_FILL0_wght200_GRAD0_opsz48.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="exit_app_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>36</width>
                 <height>36</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../../../resources.qrc">
                 <normaloff>:/icon/images/icons/close_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/icon/images/icons/close_FILL0_wght200_GRAD0_opsz24.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTabBar::tab {
    background-color: white; 
    color: black; 
	 border: 1px solid blue;
	 /*border-color: rgb(59, 107, 249);*/
	
	/*border-color: rgb(158, 158, 158);*/
	
	border-color: rgb(255, 255, 255);
	background-color: rgb(158, 158, 158);
    padding: 5px;
	border-radius: 5px; 
}
QTabBar::tab:selected {
	background-color: rgb(40, 84, 113);
	color: white;
    font-weight: bold;
	border: 3px solid gray;
	border-color: rgb(129, 194, 65);
    padding: 5px;
	border-radius: 5px; 
}

</string>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>4</number>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <widget class="QWidget" name="start">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Start</string>
          </attribute>
          <layout class="QVBoxLayout" name="start_page_layout"/>
         </widget>
         <widget class="QWidget" name="power_system_data">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Power System Data</string>
          </attribute>
          <layout class="QVBoxLayout" name="power_system_data_page_layout"/>
         </widget>
         <widget class="QWidget" name="planning_model_page">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Planning Model Setup</string>
          </attribute>
          <layout class="QVBoxLayout" name="planning_model_page_layout"/>
         </widget>
         <widget class="QWidget" name="scenario_builder_page">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Scenario Builder</string>
          </attribute>
          <layout class="QVBoxLayout" name="scenario_builder_page_layout"/>
         </widget>
         <widget class="QWidget" name="build_run">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000000</width>
            <height>1000000</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run Model&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Execute Model</string>
          </attribute>
          <layout class="QVBoxLayout" name="build_run_page_layout"/>
         </widget>
         <widget class="QWidget" name="results">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(217, 217, 217);</string>
          </property>
          <attribute name="title">
           <string>Results Viewer</string>
          </attribute>
          <layout class="QVBoxLayout" name="results_page_layout"/>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Developed by Sandia National Laboratories</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ver 1.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
